### PR TITLE
Adding a subset of tests for the migration scripts

### PR DESCRIPTION
**Pull request by maniarathi**

### CODE REVIEW

File: dcp_prototype/backend/wrangling/migrations/common/gather_dcp_one_data.py [hunk 1]
```diff
@@ -176,7 +176,6 @@ def generate_metadata_structure_from_dir(input_dir, num_threads, dataset_metadat
 
 
 def generate_metadata_structure_from_targz(input_source, dataset_metadata):
-    index = 0
```
Comment (bmccandless) [inline dcp_prototype/backend/wrangling/migrations/common/gather_dcp_one_data.py]: index is used in line  194,195.  It's used to print the number of files accessed.

File: dcp_prototype/backend/wrangling/migrations/common/metadata.py [hunk 2]
```diff
@@ -1,15 +1,9 @@
 """This module contains simple python classes that correspond to the artifact schema"""
 
-import json
-
 
 class MetadataBase(object):
     """Base class for the metadata objects"""
 
-    def to_json(self):
```
Comment (bmccandless) [inline dcp_prototype/backend/wrangling/migrations/common/metadata.py]: This function is used in migrate_dcp_one_to_dcp_two and create_artifact

File: dcp_prototype/backend/wrangling/migrations/common/gather_dcp_one_data.py (hunk 1 above)
Comment (maniarathi) [reply dcp_prototype/backend/wrangling/migrations/common/gather_dcp_one_data.py]: This variable is defined/incremented in the for loop in 188 though so this is a redundant initiation.

File: dcp_prototype/backend/wrangling/migrations/common/metadata.py (hunk 2 above)
Comment (maniarathi) [reply dcp_prototype/backend/wrangling/migrations/common/metadata.py]: I couldn't find any uses of it actually in either file; could you point me to the references please?

Comment (bmccandless) [reply dcp_prototype/backend/wrangling/migrations/common/metadata.py]: My bad, I had already refactored that part.  It's not needed.

File: dcp_prototype/backend/wrangling/migrations/common/gather_dcp_one_data.py (hunk 1 above)
Comment (bmccandless) [reply dcp_prototype/backend/wrangling/migrations/common/gather_dcp_one_data.py]: Yes, you're right.

**Pull request merged**